src/cascadia/ShellExtension/OpenTerminalHere.cpp

```diff
@@ -266,7 +266,16 @@ std::wstring OpenTerminalHere::_GetPathFromExplorer() const
         return path;
     }
 
-    auto shell = create_instance<IShellWindows>(CLSID_ShellWindows);
+    com_ptr<IShellWindows> shell;
+    try
+    {
+        shell = create_instance<IShellWindows>(CLSID_ShellWindows, CLSCTX_ALL);
+    }
+    catch (...)
+    {
+        //look like try_create_intance is not available no more
+    }

```

Comment:
You might be able to make this even easier with `try_create_instance`
```suggestion
    com_ptr<IShellWindows> shell{ try_create_instance<IShellWindows>(CLSID_ShellWindows, CLSCTX_ALL) };
```

Reply:
it looks like try_create_instance is not is the "winrt/base.h" header. or is it?

Reply:
You know what, we might not be on the version of cppwinrt that supports this. https://github.com/microsoft/cppwinrt/pull/663. So ignore me 😃